File: src/main/java/com/gmail/borlandlp/minigamesdtools/activepoints/ActivePointsCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.activepoints;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class ActivePointsCreatorHub extends CreatorHub {
    public ActivePoint createActivePoint(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (ActivePoint) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/activepoints/ActivePointsCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.activepoints

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class ActivePointsCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): ActivePoint {
        return super.create(itemID, dataProvider) as ActivePoint
    }
}